py/htm/advanced/regions/RawValues.py

```diff
@@ -0,0 +1,126 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2018, Numenta, Inc.    Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.    If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+from collections import deque
+
+from htm.bindings.regions.PyRegion import PyRegion
+from .import extractList
+
+
+class RawValues(PyRegion):
+    """
+    RawDate is a simple region used to send raw scalar values into networks.
```

Comment:
@dkeeney this sounds generally useful, do we want to promote the region from `.advanced` to `htm.regions` ? 

But that would be only for python. We already have `SensorRegion` which applies an encoder to the imputs. A solution would be to have an identity/pass-through encoder (=that does nothing with its inputs) and use that instead of this class. 
  - BaseEncoder could behave as an identity encoder.

Reply:
>  A solution would be to have an identity/pass-through encoder (=that does nothing with its inputs) and use that instead of this class.

Its the lack of a NetworkAPI region that is the problem.  This Raw region is one solution to it.  The EncoderRegion would have been able to do that if I could figure out how to make it work.

Reply:
Yes, I was thinking about a Generic encoder region #637 would come in handy here. 

So, the remaining point, do we want to move "RawValues" (maybe rename to ??) from advanced to the common py regions?

Reply:
Maybe RawSensor also. It queues sparse input and pops it as dense.
In either case, the `region.__init__.py` will need to move as well since it provides extractList.

Reply:
I am about to push the changes due to #670